dataflowAPI/rose/util/PoolAllocator.h

```diff
@@ -192,7 +192,7 @@ class PoolAllocatorBase {
             if (!freeLists_[freeListIdx]) {
                 Chunk *chunk = new Chunk;
                 freeLists_[freeListIdx] = chunk->fill(cellSize_);
-                SAWYER_THREAD_TRAITS::LockGuard lock(chunkMutex_);
+                SAWYER_THREAD_TRAITS::LockGuard chunkLock(chunkMutex_);
                 chunks_.push_back(chunk);
```

Comment:
Is the rename lock -> chunkLock intentional?

Reply:
Intentional to eliminate shadowed name